tests/integration/wordpress_client.py

```diff
@@ -118,7 +118,7 @@ def _create_post(self, title: str, content: str):
         )
         return response.json()
 
-    def _create_comment(self, post_id: int, post_link: str, content: str):
+    def create_comment(self, post_id: int, post_link: str, content: str):
         """Add a comment to a WordPress post using HTML form, return url link of the new comment"""
```

Comment:
Do you think it is worth updating the docstring to our new standard and to define a return type?